src/styles/components/_switchtheme.scss

```diff
@@ -1,35 +1,86 @@
 .switch-theme {
   display: flex;
-  justify-content: center;
-  align-items: center;
+  margin: 0 30px 0 30px;
```

Comment:
Maybe removing the left margin in very low width?

Reply:
```suggestion
  margin: 0 30px;
```